packages/create-sitecore-jss/src/templates/nextjs-sxa/src/lib/next-config/plugins/robots.js

```diff
@@ -0,0 +1,16 @@
+const robotsPlugin = (nextConfig = {}) => {
+  return Object.assign({}, nextConfig, {
+    async rewrites() {
+      // When disconnected we proxy to the local faux layout service host, see scripts/disconnected-mode-proxy.ts
+      return [
```

Comment:
This would override the rewrites defined in base plugins. The code here should check if `nextConfig` param has `rewrites` defined and append to those if so.

Also, the comment here looks to be a c/p artifact and should be updated or removed.